Data/Localizations/ru.lua

```diff
@@ -914,8 +920,8 @@ L.SOFTRES_TOOLTIP_HARD_RESERVED_NOTE = "|c00CC2743 Примечание: %s";
 L.SOFTRES_TOOLTIP_RESERVED_BY = "Зарезервировано";
 L.SOFTRES_WHISPER_PREFIXES = "!sr|!softres|!softreserve";
 L.SOMETHING_WENT_WRONG_WARNING = "Что-то пошло не так!";
-L.START = "Начинать";
-L.STOP = "Останавливаться";
+L.START = "   Старт   "; -- KartonArmadon
```

Comment:
What is the reason for these? Since Start also only has 5 characters I can't imagine there being a spacing issue right?